adversary/repeat.go

```diff
@@ -0,0 +1,64 @@
+package adversary
+
+import (
+	"github.com/filecoin-project/go-f3/gpbft"
+)
+
+type (
+	Repeat struct {
+		id            gpbft.ActorID
+		host          gpbft.Host
+		echoCountDist CountSampler
+	}
+	CountSampler interface {
+		// TODO replace with math/rand/v2 rand.Source once upgraded to go 1.22.
+		Uint64() uint64
```

Comment:
Can you document this? What does the return value mean? It's not clear how I can make one of these that:
- echoes every message once
- echoes some fraction of messages at most once
- echoes every messages a <distribution> number of times
- echoes some fraction of messages <distribution> number of times

... or if some of these are not possible.